modules/ROOT/pages/audit-logging.adoc

```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
+| Subaction: none + 
+
+Properties:
+
+* Process name
+* Process phase
+* Applications required
+** Application name
+* Process team
+** Member name
+** Assigned phases
+* Advanced permissions option
+
+
+| Edit process
+| Process
+| Process ID
+| N/A
+| Edit
+| Subaction: none + 
+
+Properties:
+
+* Process name
+* Process phase
+* Applications required
+** Application name
+* Process team
+** Member name
+** Assigned phases
+* Advanced permissions option
+
+| Delete process
+| Process
+| Process ID
+| N/A
+| Delete
+| Subaction: none + 
+
+Properties:
+
+* Process name
+* Process phase
+
+|===
+
+
+=== Bots 
+
+
+[%header,cols="25a,15a,15a,15a,15a,30a"]
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+| Register bot
+| Bot
+| Bot ID
+| N/A
+| Register 
+| Subaction: none + 
+
+Properties:
+
+* Bot name
+* Api key name
+* Bot version
+* IP address
+* Hostname
+* Time zone
+
+
+
+| Register bot
+| Bot
+| Bot ID
+| N/A
+| Register 
+| Subaction: none + 
+
+Properties:
+
+* Bot name
+* Api key name
+* Bot version
+* IP address
+* Hostname
+* Time zone
+* Bot status
+* Assigned applications
+** Application name
+* Assigned service times
+** Service time name
+* Phase affiliation
+* Description
+* Assigned Secure Sessions
+
+
+| Delete bot
+| Bot
+| Bot ID
+| N/A
+| Delete 
+| Subaction: none + 
+
+Properties:
+
+* Bot name
+
+|===
+
+
+=== Credential Pool
+
+
+[%header,cols="25a,15a,15a,15a,15a,30a"]
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+
+| Create credential
+| Credential
+| Credential ID
+| N/A 
+| Create 
+| Subaction: none + 
+
+Properties:
+
+* Credential name
+* Credential description
+* User affiliation
+* User group affiliation
+* Phase affiliation
+* Type
+
+
+| Edit credential
+| Credential
+| Credential ID
+| N/A 
+| Edit 
+| Subaction: none + 
+
+Properties:
+
+* Credential name
+* Credential description
+* User affiliation
+* User group affiliation
+* Phase affiliation
+
+
+| Delete credential
+| Credential
+| Credential ID
+| N/A 
+| Delete 
+| Subaction: none + 
+
+Properties:
+
+* Credential name
+
+
+|===
+
+
+=== Global Variables
+
+
+[%header,cols="25a,15a,15a,15a,15a,30a"]
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+
+| Create global variable
+| Global variable
+| Global variable ID
+| N/A
+| Create
+| Subaction: none + 
+
+Properties:
+
+* Global variable name
+* Global variable description
+* Phase affiliation
+* Type
+
+
+| Edit global variable
+| Global variable
+| Global variable ID
+| N/A
+| Edit
+| Subaction: none + 
+
+Properties:
+
+* Global variable name
+* Global variable description
+* Phase affiliation
+
+| Delete global variable
+| Global variable
+| Global variable ID
+| N/A
+| Delete
+| Subaction: none + 
+
+Properties:
+
+* Global variable name
+
+
+|===
+
+
+=== Applications 
+
+
+[%header,cols="25a,15a,15a,15a,15a,30a"]
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+
+| Create Application
```

Comment:
```suggestion
| Create application
```

Reply:
Not sure if these are UI elements. If they are capitalized then use the same name.

Reply:
This is kind of a UI element, so I left it upper case (It's not an application but a kind of tag for which applications the bot can access)